src/index.html

```diff
@@ -9,6 +9,15 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="wrap">
+      <div class="clock">
+        <div class="clock__seconds-arrow"></div>
+        <div class="clock__minure-arrow"></div>
+        <div class="clock__center-dot"></div>
+      </div>
+    </div>
+
+
+
   </body>
```

Comment:
```suggestion
    </div>
  </body>
```
don't add spaces between child and parent elements